cmd/kubeadm/app/discovery/token/token.go

```diff
@@ -27,44 +34,81 @@ import (
 	clientcmdapi "k8s.io/client-go/tools/clientcmd/api"
 	"k8s.io/kubernetes/cmd/kubeadm/app/constants"
 	kubeconfigutil "k8s.io/kubernetes/cmd/kubeadm/app/util/kubeconfig"
+	"k8s.io/kubernetes/cmd/kubeadm/app/util/pubkeypin"
 	tokenutil "k8s.io/kubernetes/cmd/kubeadm/app/util/token"
 	bootstrapapi "k8s.io/kubernetes/pkg/bootstrap/api"
 	"k8s.io/kubernetes/pkg/controller/bootstrap"
 )
 
 const BootstrapUser = "token-bootstrap-client"
 
+var (
```

Comment:
I'm ok with it, but we aren't doing it in other places. This seems to be a good place to use this method though...